app/views/users/index.html.erb

```diff
@@ -3,17 +3,24 @@
 <html>
 <head>
   <title>My Blog</title>
-  <link rel="stylesheet" type="text/css" href="styles.css">
+  <link rel="stylesheet" type="text/css" href="styles.css"> 
 </head>
 <body>
   <header>
-    <h1>Welcome to My Blog</h1>
+    <h1>Blog Posts Home</h1>
   </header>
 
   <main>
     <h2>Users</h2>
   </main>
 
+  <% @users_with_posts_count.each do |user| %>
+    <div>
+      <h2><%= link_to user.name, user %></h2>
+      <p>Posts count: <%= user.posts_count %></p>
+    </div>
+  <% end %>
+  
```

Comment:
- [ ] It is nice to keep to design guidelines, kindly match your UI to the given wireframe
in your design :point_down: 
![Screenshot from 2023-11-25 11-42-42](https://github.com/paulshaun-kdtk/blog-posts/assets/66097371/b9ceadab-8cb2-4805-98e9-701671b8a9df)
given wireframe :point_down: 
![Screenshot from 2023-04-21 17-06-41](https://github.com/paulshaun-kdtk/blog-posts/assets/66097371/8a127332-97cd-4e9c-8761-50b92c621104)